app/src/main/java/com/hadiyarajesh/notex/ui/settings/SettingsItems.kt

```diff
@@ -0,0 +1,45 @@
+package com.hadiyarajesh.notex.ui.settings
+
+import androidx.annotation.DrawableRes
+import com.hadiyarajesh.notex.R
+
+sealed class SettingsItems(
```

Comment:
As we already have `sealed class Screens`
Why to use a separate `SettingsItems`?

Reply:
updated in the next PR maybe tomorrow.